Wave1_data/4.processing_profiled_features/scripts/5.aggregate_profiles.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
```

Comment:
Confirm this is ran in notebook prior to nbconvert.

Reply:
Definitely not. I will make a note of why - RAM. needs to run on cluster